features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/tenant/tenant-conf.json

```diff
@@ -1,5 +1,29 @@
 {
   "EnableMonetization" : false,
   "IsUnlimitedTierPaid" : false,
-  "ExtensionHandlerPosition": "bottom"
+  "ExtensionHandlerPosition": "bottom",
+  "RESTAPIScopes": {
+    "Scope": [
+      {
+        "Name": "API_PUBLISHER_SCOPE",
+        "Roles": "admin"
+      },
+      {
+        "Name": "API_CREATOR_SCOPE",
+        "Roles": "admin"
+      },
+      {
+        "Name": "API_CREATOR_PUBLISHER_SCOPE",
+        "Roles": "admin"
+      },
+      {
+        "Name": "API_SUBSCRIBER_SCOPE",
+        "Roles": "Internal/subscriber"
```

Comment:
Why this is special case? IMO we should use admin for all and let users to figure out what they exactly need.